src/components/Stop.jsx

```diff
@@ -2,12 +2,12 @@ import classNames from './Stop.module.css'
 
 export default function Stop ({ name, children }) {
   return (
-    <div className={classNames['stop-container']}>
-      <div className={classNames['stop-header']}>
+    <article className={classNames['stop-container']}>
```

Comment:
This is a stretch, but I think our use here _does_ fall under the `article` criteria technically